code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -71,9 +71,10 @@
 /mob/living/carbon/monkey/updatehealth()
 	. = ..()
 	var/slow = 0
-	var/health_deficiency = (100 - health)
-	if(health_deficiency >= 45)
-		slow += (health_deficiency / 25)
+	if(!has_trait(TRAIT_IGNOREDAMAGESLOWDOWN))
+		var/health_deficiency = (100 - health)
+		if(health_deficiency >= 45)
+			slow += (health_deficiency / 25)
 	add_movespeed_modifier(MOVESPEED_ID_MONKEY_HEALTH_SPEEDMOD, TRUE, 100, override = TRUE, multiplicative_slowdown = slow)

```

Comment:
movespeed isn't touched if you have trait, move under

Reply:
as with slime